frontend/.env.template.txt

```diff
@@ -0,0 +1,8 @@
+# In .env
```

Comment:
please rename this to .env.template without the txt ...

Reply:
sure - didn't realize that Windows / the editor I used had added this..